src/Font.elm

```diff
@@ -128,6 +152,44 @@ fontStyleProperty isItalic =
     if isItalic then "italic" else "normal"
 
 
+-- ANDROID
+
+type alias AndroidFont =
+    String
+
+android : List Font -> List AndroidFont
+android fonts =
+    groupByFamily fonts
+        |> Dict.values
+        |> List.map androidFamily
+
+androidFamily : List Font -> AndroidFont
+androidFamily fonts =
+    "<?xml version=\"1.0\" encoding=\"utf-8\"?>"
+        ++ "\n"
+        ++ "<font-family xmlns:app=\"http://schemas.android.com/apk/res-auto\">"
+        ++ "\n"
+        ++ String.join "\n" (List.map androidFont fonts)
+        ++ "\n"
+        ++ "</font-family>"
+        ++ "\n"
+
+androidFont : Font -> String
+androidFont { weight, isItalic, ttf } =
+    "    <font"
+        ++ "\n        "
+        ++ "android:fontStyle=\"" ++ fontStyleProperty isItalic ++ "\""
+        ++ "\n        "
+        ++ "android:fontWeight=\"" ++ weightToString weight ++ "\""
+        ++ "\n        "
+        ++ "android:font=\"" ++ "@font/" ++ androidFilename ttf ++ "\""
+        ++ " />"
+
+androidFilename : String -> String
+androidFilename =
+    String.map (\c -> if c == '-' then '_' else Char.toLower c)
```

Comment:
https://package.elm-lang.org/packages/elm/core/latest/String#replace :-)

This might be more simply written as something like:

```
  String.toLower >> String.replace "-" "_"
```

Reply:
I was looking for `String.replace` in the docs! I must have just scrolled straight past it... 🙈